Extractor/Config.cs

```diff
@@ -642,7 +640,43 @@ public class SubscriptionConfig
         /// Log bad subscription datapoints
         /// </summary>
         public bool LogBadValues { get; set; } = true;
+
+        /// <summary>
+        /// Server namespace nodes the should be subscribed to a rebrowse upon changes to their values.
+        /// </summary>
+        public RebrowseTriggersConfig? RebrowseTriggers { get; set; }
     }
+    public class RebrowseTriggersConfig
+    {
+        public RebrowseTriggersTargets Targets = 
+            new RebrowseTriggersTargets();        
+        /// <summary>
+        /// A list of namespaces filters
+        /// </summary>
+        public IEnumerable<string> Namespaces { get; set; } = new List<string>();
+    }
+
+    public class RebrowseTriggersTargets 
+    {
+        private List<string> _toBeSubscribed = new List<string>();
+
+        public bool NamespacePublicationDate { 
+            get => _toBeSubscribed.Contains("NamespacePublicationDate");
+            set {
+                var exists = _toBeSubscribed.Contains("NamespacePublicationDate");
+
+                if (value && !exists) {
+                    _toBeSubscribed.Add("NamespacePublicationDate");
+                } else if (!value && exists) {
+                    _toBeSubscribed.Remove("NamespacePublicationDate");
+                }
+            }
+        }
+
+        // TODO: rename this method appropriately.
+        public List<string> GetValues => _toBeSubscribed;
```

Comment:
Two minor things here. This project does not use `_` prefixes for private members, (I know it's convention, but consistency trumps). Also it is common to use a similar name for the public property as the private member, to make the relationship clear, i.e. this should probably be `ToBeSubscribed`.